searx/engines/bitchute.py

```diff
@@ -0,0 +1,58 @@
+# SPDX-License-Identifier: AGPL-3.0-or-later
+"""bitchute (Videos)"""
+
+from json import dumps
+from datetime import datetime
+from searx.utils import html_to_text
+
+about = {
+    "website": 'https://bitchute.com',
+    "wikidata_id": "Q45287179",
+    "official_api_documentation": None,
+    "use_official_api": False,
+    "require_api_key": False,
+    "results": "JSON",
+}
+
+base_url = "https://api.bitchute.com/api/beta/search/videos"
+categories = ['videos']
+paging = True
+
+
+def request(query, params):
+
+    start_index = (params["pageno"] - 1) * 50
+    data = {"offset": start_index, "limit": 50, "query": query, "sensitivity_id": "normal", "sort": "new"}
```

Comment:
Please move 50 to a global `results_per_page` variable.

Also, 50 seems quite high, we usually set it to `10` or `20` per engine.